inference-engine/src/mkldnn_plugin/nodes/mkldnn_quantize_node.h

```diff
@@ -74,6 +75,15 @@ class MKLDNNQuantizeNode : public MKLDNNNode {
     std::vector<float> outputScale;
     std::vector<float> outputShift;
 
+    // mkldnn style post ops data representation
+    bool isPostOpDataInitialized = false;
+    mkldnn::impl::shifts_t<float> cropLowData;
+    mkldnn::impl::shifts_t<float> cropHighData;
+    mkldnn::impl::scales_t inputScaleData;
+    mkldnn::impl::shifts_t<float> inputShiftData;
+    mkldnn::impl::scales_t outputScaleData;
+    mkldnn::impl::shifts_t<float> outputShiftData;
```

Comment:
Is it safe here to keep pointers on class members in external object? Are you sure that they will not be accessed or double freed after MKLDNNQuantizeNode instance deletion?

Reply:
In this case I am sure: they are used inside mkldnn and it doesn't call delete operator for this pointers. In general this is common interface of communication between plug-in and mkldnn: we pass memory that stored inside CPU plug-in structures (actually MKLDNNMemory) through const pointers into mkldnn.